examples/js-cdn-typescript/README.md

```diff
@@ -0,0 +1,49 @@
+# Web app with Javascript CDN
+
+This is a basic web application written in TypeScript that uses RudderTyper client library along with JS CDN script.
+
+## Steps to run the sample app
+
+1. Run `npm run setup` to install the required dependencies.
+2. Replace `writeKey` and `dataPlaneUrl` values in `./public/index.html` file.
+3. Run the command `npm start`. The app will run in development mode. Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+4. Click on the track button to fire event.
+
+![Alt text](app.png?raw=true 'Sample Site')
+
+## Regenerate client libraries
+
+1. Run the command `npx rudder-typer`. This will generate new `index.ts` and `rudder.ts` file based on the ruddertyper.yml.
+2. Run the command `browserify src/analytics/index.ts -p [ tsify ] --standalone rudderTyper > public/rudderTyperBundle.js` to generate the browserify bundle.
```

Comment:
Please add this step to the development section as well.

Reply:
Done